test/migrations.unit.spec.js

```diff
@@ -43,6 +43,7 @@ const configs = [
 
 const excludedForms = new Set([
   '28-1900',
+  '21-526EZ',
```

Comment:
@cehsu LMK if this is the proper usage of this list. This form was added to the `vets-json-schema` recently but hasn't been added to the `vets-website` yet which is causing a couple of unit tests to fail.

Reply:
This is correct @rianfowler. I'll add documentation to that test, sorry about the lack of clarity.